llvm/test/CodeGen/AMDGPU/mem-builtins.ll

```diff
@@ -36,7 +34,6 @@ define amdgpu_kernel void @test_memchr(ptr addrspace(1) %src, i32 %char, i64 %le
 ; GCN: s_addc_u32 s{{[0-9]+}}, s{{[0-9]+}}, strcpy@rel32@hi+12
 define amdgpu_kernel void @test_strcpy(ptr %dst, ptr %src) #0 {
   %res = call ptr @strcpy(ptr %dst, ptr %src)
-  store ptr %res, ptr addrspace(1) undef
```

Comment:
Really for all of these, should stop storing to undef. The point is to have a dummy use of the value, it doesn't matter what it is